### PR TITLE
add .sass formatting

### PR DESCRIPTION
**Pull request by TheRealSyler**
should pass the tests and be ready to merge.

related to #1433

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -326,6 +326,19 @@
           ],
           "description": "Default formatter for <style lang='scss'> region"
         },
+        "vetur.format.defaultFormatter.sass": {
+          "type": "string",
+          "default": "sass",
+          "enum": [
+            "none",
+            "sass"
```
Comment (octref) [inline package.json]: Do you mind making this `sass-formatter`? `sass` is the general language name and I doubt people can find your work/repo of `sass-formatter`.

File: server/src/modes/style/sass/sassLanguageMode.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    return 'sass';
+  }
+
+  configure(c: any) {
+    this.config = c;
+  }
+
+  doValidation(document: TextDocument): any {
+    return undefined;
+  }
```
Comment (octref) [inline server/src/modes/style/sass/sassLanguageMode.ts]: All these methods are optional — why do you have to implement them?

Comment (TheRealSyler) [reply server/src/modes/style/sass/sassLanguageMode.ts]: i don't remember why, i will have a look at it.

**Comment by octref** [comment]
Thanks!

**Pull request merged**